yodeploy/cmds/tests/test_build_artifact.py

```diff
@@ -7,19 +7,19 @@
 class TestGitHelper(TestCase):
 
     def setUp(self):
-        self.patcher = patch('yodeploy.cmds.build_artifact.check_output')
+        self.patcher = patch('subprocess.check_output')
         self.check_output = self.patcher.start()
         self.git = GitHelper()
 
     def cleanUp(self):
         self.patcher.stop()
 
     def test_uses_get_show_for_the_commit_message(self):
-        self.check_output.return_value = 'my commit message'
+        self.check_output.return_value = b'my commit message'
         self.assertEqual(self.git.commit_msg, 'my commit message')
         git_call = self.check_output.call_args[0][0][0:2]
         self.assertEqual(git_call, ('git', 'show'))
 
     def test_removes_special_chars_from_the_commit_message(self):
-        self.check_output.return_value = 'frosty the ☃'
+        self.check_output.return_value = b'frosty the \xe2\x98\x83'
```

Comment:
If we drop support for Python 3.2, we can do the far more readable: `u'frosty the ☃'.encode('utf-8')`